recipe/meta.yaml

```
@@ -13,34 +13,41 @@ build:
   entry_points:
     - ydata_profiling = ydata_profiling.controller.console:main
     - pandas_profiling = ydata_profiling.controller.console:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
   number: 0
+  skip: True   #  [py<37]
 
 requirements:
   host:
-    - python >=3.7,<3.11
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7,<3.11
-    - scipy >=1.4.1,<1.10
+    - python
+    - scipy=1.9.3
```

Curious why the scipy pinnings were changed?
 https://github.com/ydataai/ydata-profiling/blob/v4.1.1/requirements.txt#LL1-L1C20
 ```suggestion
    - scipy >=1.4.1,<1.10
```

---

Hi Eric, there were a few issues raised regarding this in the [upstream](https://github.com/ydataai/ydata-profiling/issues/1284) where the requirements had a restriction of `scipy < 1.10` in which for our defaults, we did not have scipy version less than 1.10 was built for compatibility for python 3.11 which therefore caused build failures for this package that is needed for scikit-image 0.20.0

---

So scipy 1.9.3 had to be built for python 3.11 and uploaded to defaults specifically in support of this package